src/tableStyle.js

```diff
@@ -1,109 +1,119 @@
 import { makeStyles } from '@eqworks/lumen-labs'
 
-const tableStyle = makeStyles(({
+const tableStyle = ({ headerTitle, centerHeader, compactTable }) => makeStyles(({
```

Comment:
Would recommend naming the module as `src/table-style.js` to stay consistent with the rest of the conventions